samples/speech/index.html

```diff
@@ -46,94 +42,99 @@
 
         <div id="BotChatGoesHere"></div>
 
-        <script src="../../botchat.js"></script>        
-        <!--If you do not want to use Cognitive Services lib, comment out the following line -->
-        <script src="../../CognitiveServices.js"></script>
-        
-        <script>
+        <script src="../../botchat.js" type="text/javascript"></script>
+
+        <!-- If you do not want to use Cognitive Services library, comment out the following line -->
+        <script src="../../CognitiveServices.js" type="text/javascript"></script>
+
+        <script type="text/javascript">
             var params = BotChat.queryParams(location.search);
 
             var user = {
-                id: params['userid'] || 'userid',
-                name: params["username"] || 'username'
-                };
-            
+                id: params.userid || 'userid',
+                name: params.username || 'username'
+            };
+
             var bot = {
-                id: params['botid'] || 'botid',
-                name: params["botname"] || 'botname'
+                id: params.botid || 'botid',
+                name: params.botname || 'botname'
             };
-            
-            window['botchatDebug'] = params['debug'] && params['debug'] === "true";
 
-            // // Option 1 : No Speech
+            window.botchatDebug = params.debug && params.debug === 'true';
```

Comment:
This is obviously correct JavaScript, but the reason I prefer the `[]` style is that this will choke in TypeScript. This is not currently TypeScript, but I'd like the option for people (including myself) to copy and past this into TypeScript code.